hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/Utils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.fixtures;
+
+import java.util.UUID;
+
+public class Utils {
+    public static byte[] randomUtf8Bytes(int n) {
+        byte[] data = new byte[n];
+        int i = 0;
+        while (i < n) {
```

Comment:
Curious, why did we write this instead of using SecureRandom?
```
SecureRandom.getInstanceStrong().nextBytes(data);
```
Is SecureRandom not random enough?

Reply:
No, I think I can use `SecureRandom`. This was a method in `mono-service`. I re-used it and moved to testFixtures.

Reply:
Just saw that there is a randomBytes method in TestBase, will use that.